src/navigator/DrawerNavigator.tsx

```diff
@@ -283,7 +285,20 @@ export default function DrawerNavigator() {
           name={Screens.BackupIntroduction}
           // @ts-expect-error component type in native-stack v6
           component={BackupIntroduction}
-          options={{ title: t('accountKey'), drawerIcon: AccountKey }}
+          options={{
+            drawerLabel:
+              !backupCompleted && getOnboardingExperimentParams().showBackupAlert
+                ? () => (
+                    <Text style={styles.itemStyle}>
+                      {t('accountKey')}
+                      <View style={styles.drawerItemIcon}>
+                        <ExclamationCircleIcon />
```

Comment:
not a big deal but it looks like in the designs the vertical center line of the text is aligned with the center of the icon, whereas here, it looks like the bottom line of the text is aligned with the bottom of the icon

Reply:
fixed